iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties.template

```diff
@@ -242,6 +242,27 @@ dn_data_dirs=data/datanode/data
 # Datatype: String
 dn_multi_dir_strategy=SequenceStrategy
 
+# load_disk_select_strategy
+# The strategy is used to choose a proper disk for loading tsfile.
+# The info of the two strategies are as follows:
+# 1. MIN_IO_FIRST: the system will try to choose the same disk as tsFileToLoad for loading.
+# 2. DISK_STORAGE_BALANCE_FIRST: the system will make the disk choice that balance storage as much as possible.
```

Comment:
In fact, for the DISK_STORAGE_BALANCE_FIRST policy, which applies the storage engine's default multi-disk policy, this is not necessarily the balancing policy, so it feels like we need to rename this and update the description.